### PR TITLE
Empyrical - financial risk analysis

### PR DESCRIPTION
**Pull request by alen12345**

### CODE REVIEW

**Comment by alen12345** [comment]
@CJ-Wright please tell me which comments you want removed to finalize the pull request.

File: recipes/empyrical/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - bottleneck
+
+test:
+  imports:
+    - empyrical
+    - empyrical.tests
+
+about:
+  home: https://github.com/quantopian/empyrical
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
```
Comment (CJ-Wright) [inline recipes/empyrical/meta.yaml]: Please remove this comment

File: recipes/empyrical/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+test:
+  imports:
+    - empyrical
+    - empyrical.tests
+
+about:
+  home: https://github.com/quantopian/empyrical
+  license: Apache 2.0
+  license_family: Apache
+  # TODO - license_file: LICENSE
```
Comment (CJ-Wright) [inline recipes/empyrical/meta.yaml]: Was the license not released with the code?

Comment (alen12345) [reply recipes/empyrical/meta.yaml]: No it wasn't...in fact, I am going to ask for it to be included in the next release on pypi. Can we go on without for the time being and update the conda-forge package with the license as soon as we push that fix on pypi?

Comment (CJ-Wright) [reply recipes/empyrical/meta.yaml]: The Apache license requires that the license be in the package. You can package it manually https://conda-forge.org/docs/meta.html#packaging-the-license-manually

**Comment by CJ-Wright** [comment]
@alen12345 Welcome to Conda-Forge!

**Pull request merged**